src/components/Exchange/ExchangeTVChart.js

```diff
@@ -464,13 +314,18 @@ export default function ExchangeTVChart(props) {
         </div>
       </div>
       <div className="ExchangeChart-bottom App-box App-box-border">
-        <div className="ExchangeChart-bottom-header">
-          <div className="ExchangeChart-bottom-controls">
-            <Tab options={Object.keys(CHART_PERIODS)} option={period} setOption={setPeriod} />
-          </div>
-          {candleStatsHtml}
-        </div>
-        <div className="ExchangeChart-bottom-content" ref={chartRef}></div>
+        {chartToken.symbol && chainId && (
+          <TVChartContainer
+            currentPositions={currentPositions}
+            currentOrders={currentOrders}
```

Comment:
if i understand correctly idea is to have separate `TVChartContainer` component without domain logic related to orders/positions to keep it pure, e.g. it does not calculate liquidation prices for positions

i think it can accept prop `lines` (or probably some better name) instead of currentPositions/currentOrders:
* TVChartContainer would be more "pure"
* currentPosition and currentOrders are almost the same, so it would remove some duplication

Reply:
pinging in case this comment was missed

Reply:
Sorry I missed this one. This one is really good idea. I have updated the code based on this.